src/sidepanel/v2/components/MessageItem.tsx

```
@@ -167,6 +167,97 @@ const SelectedTabDataDisplay = ({ content }: SelectedTabDataDisplayProps) => {
+
+const urlRegex = /https?:\/\/[^\s)>,]+/g
+
+const sanitizeUrl = (u: string): string => {
+  return u.replace(/[),]+$/g, '')
+}
+
+const parseExtractedLinks = (content: string): ExtractedLink[] => {
+  const results: ExtractedLink[] = []
+  if (!content) return results
+  const labelRegex = /([A-Z][A-Za-z0-9 .&-]{1,50}):/g
+  const segments: Array<{ label?: string, text: string }> = []
```

ESLint

[error] 'segments' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)